en/user_guide/transverse.rst

```diff
@@ -0,0 +1,27 @@
+Global page or transverse node
+==============================
+
+Each singular website has a global page (node transverse) per languages, it is a node containing blocks common to the other pages.
+This node allows to configure blocks, those blocks which would be available in all the other pages of this website.
+
+Global pages have several languages, don't have a `workflow`_ and have only one version.
+
+Transverse block
+----------------
+
+Transverse blocks are blocks in the area of the transverse node, these blocks are common to the other pages.
+The block configuration can only be done in the transverse node
+
+To add a transverse block in the transverse node select a block in the block list, drag and drop it on the area.
```

Comment:
Ok you're talking about the blocks panel to contribute a standard node, I think you should mention it at the start of the paragraph, it's a little bit hard to get the context here :)